src/plugins/discover/public/application/view_components/utils/use_search.ts

```diff
@@ -432,7 +431,7 @@ export const useSearch = (services: DiscoverViewServices) => {
       }
 
       filterManager.setAppFilters(actualFilters);
-      data.query.queryString.setQuery(savedQuery ? data.query.queryString.getQuery() : query);
+      data.query.queryString.setQuery(query);
```

Comment:
I removed this here because originally, `query` was either the query from the `savedSearch` or the default query. Since I changed `query` to `data.query.queryString.getQuery()`, it shouldn't have any impact since we will instead only be using `data.query.queryString.getQuery()` if there is no saved search.